### ISSUE TITLE
CSS layout issues when viewing entries backend

### ISSUE THREAD

**Issue by JiveDig** [Desertsnowman/Caldera-Forms#287]
Not a huge deal, but the 2 columns should probably jump to full width, or similar, on smaller screens so the entries table doesn't have weird wrap issues.
<img width="1107" alt="screen_shot_2015-10-18_at_8_03_33_pm" src="https://cloud.githubusercontent.com/assets/540990/10567474/eed22e8a-75d3-11e5-948b-aaa4e17fc0df.png">
<img width="852" alt="screen shot 2015-10-18 at 8 05 07 pm" src="https://cloud.githubusercontent.com/assets/540990/10567475/f1c1ca92-75d3-11e5-836a-6779e80a3674.png">

**Comment by paaljoachim** [comment]
It would be nice to get rid of the quick edit entries in the forms screen. Just keeping entries in its own section.

**Comment by Shelob9** [comment]
I agree. Pretty sure their is an open issue on this, but yah, the list
table for forms should minimize with a big old close button for entry
viewer to get it back.

--
JoshPress.net <http://joshpress.net/>
CalderaWP.com <http://calderawp.com/>
IngotHQ.com
@Josh412 <http://twitter.com/Josh412>


On 24 December 2015 at 08:19, Paal Joachim Romdahl <notifications@github.com
